### DISCUSSION TITLE
Masked Textbox in 3.4.1

### DISCUSSION THREAD

**Discussion by charesch** [HandyOrg/HandyControl#1216] [Q&A] [answered]
Hi,

I had an application with MaskedTextbox's but I think they don't exist in 3.4.1. Can someone guide me in replicating the behavior of a masked text box with current controls? I want to guide the user within the formating ie a phone number "(___) ___-____".

Thanks for your time!
Christian

**Comment by ghost1372** [comment, marked as answer]
MaskedTextBox is removed in 3.4.1. You should use older version
